README.md

```diff
@@ -0,0 +1,38 @@
+# Actors
+
+Actors are Filecoin’s notion of smart contracts. 
+They are not true smart contracts—with bytecode running on a VM—but instead implemented in Go. 
+It is expected that other implementations will match the behaviour of the Go actors exactly. 
+An ABI describes how inputs and outputs to the VM are encoded. 
+Future work will replace this implementation with a "real" VM.
+
+The [Actor](https://github.com/filecoin-project/lotus/blob/master/chain/types/actor.go) struct is the base implementation of actors, with fields common to all of them.
```

Comment:
IMO, it's a little weird for specs-actors to refer back to an implementation. Maybe rephrase this to say something like "see, for example, <Lotus actor link> and <go-filecoin actor link>"?

Reply:
Yes, remove this. The linked concept does not exist in this repo, but is abstracted. Ditto for the four bullet points below. In this repo those are better thought of as abstractions provided by the runtime, but it's not nearly a complete list of those abstractions so better to omit it. Linking to the [runtime.go](https://github.com/filecoin-project/specs-actors/blob/master/actors/runtime/runtime.go) file would be better - it has decent doc-comments.